### PR TITLE
Feat/add form request

### PR DESCRIPTION
**Pull request by rayfajars**
## OVERVIEW

- Slicing Form Request
- add response succes and error if form sumbitted 
- add watch for show and hide evidance

preview :
image

## TASK 
[FORM REQUEST](https://app.clickup.com/t/301b37b)

## EVIDANCE
title : slicing and form request and add reposne 
project : Digiteam Inventaris Platform
participants: @rayfajars @Ibwedagama @yoslie @doohanas @agunghide

### CODE REVIEW

File: src/components/RequestComponents/FormRequest.vue [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+                text: "Berhasil mengirim permohonan",
+                icon: "success",
+                confirmButtonText: "OK",
+              }).then(() => {
+                // todo : redirect to list request page
+                // this.$router.push("/list-request");
+              });
+            })
+            .catch((err) => {
+              console.log(err);
```
Comment (Ibwedagama) [inline src/components/RequestComponents/FormRequest.vue:55]: minor:

please remove the console statement @rayfajars 👍🏼

**Review by Ibwedagama — APPROVED** [review]

File: src/components/RequestComponents/FormRequest.vue (hunk 1 above)
Comment (rayfajars) [reply src/components/RequestComponents/FormRequest.vue]: [done bli](https://github.com/jabardigitalservice/digiteam-inventaris-app/pull/2/commits/8a0ff81622593a11102efcaeff7b63de569c93a1)

**Review by agunghide — APPROVED** [review]

**Review by yoslie — APPROVED** [review]
discussion:
Maybe we can use another option for using `axios` in this project by create a plugin and assign a global `axios` instance. So we don't need to import it whenever it's needed.

File: src/components/RequestComponents/FormRequest.vue [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+        divisi: "",
+        item_name: "",
+        purpose: "",
+        urgency: "",
+      },
+    };
+  },
+  methods: {
+    handleSubmit() {
+      console.log(this.formRequest);
```
Comment (doohanas) [inline src/components/RequestComponents/FormRequest.vue]: Hello @rayfajars, Please remove this console too 👍🏽

**Review by doohanas — APPROVED** [review]

File: src/components/RequestComponents/FormRequest.vue (hunk 2 above)
Comment (rayfajars) [reply src/components/RequestComponents/FormRequest.vue]: in the last update i done , remove this console log

you can check in the last commit [here](https://github.com/jabardigitalservice/digiteam-inventaris-app/pull/2/commits/8a0ff81622593a11102efcaeff7b63de569c93a1)

and view file

**Pull request merged**